nemo_skills/inference/server/model.py

```diff
@@ -165,14 +167,18 @@ def __call__(
         if not self.handle_code_execution:
             request["prompts"] = prompts
             outputs = self._single_call(**request)
-            outputs = [
-                {
-                    'generated_solution': remove_stop_tokens(output, stop_phrases),
-                    'predicted_answer': extract_answer(output),
-                    'error_message': extract_error_message(output),
-                }
-                for output in outputs
-            ]
+            # if process_output is False, return the raw outputs
```

Comment:
Can you explain a bit more what exactly you need from the output? While we can certainly add this flag, I'm trying to understand better what's the use-case is and whether there is another way to get to what you want. The problem is that those fields are also used in other scripts, so basically disabling process_output is guaranteed to break anything else you do with our pipeline and while it might be fine for your case, since it seems that you just want to run inference and then not use anything else, it might be confusing for other cases, where someone might enable this flag and still expect the other scripts to work. Plus, there are other places where the inference can happen and so we should make sure this flag is available there as well.

Happy to sync offline as well if that's easier.